imagedephi/gui.py

```diff
@@ -12,9 +12,12 @@
 def select_directory(request: Request, path: str = "/"):
     base_path = pathlib.Path(path)
 
-    if not base_path.is_dir():
+    if not base_path.exists():
```

Comment:
[`Path.is_dir`](https://docs.python.org/3/library/pathlib.html#pathlib.Path.is_dir) states:
> `False` is also returned if the path doesn’t exist or is a broken symlink

Why is a separate check for both `.exists()` and `.is_dir()` necessary?

Reply:
It's not. I was mistaken.